tests/tensorflow/conftest.py

```diff
@@ -94,3 +97,45 @@ def weekly_tests(request):
 @pytest.fixture(scope="module")
 def models_dir(request):
     return request.config.getoption("--models-dir")
+
+
+@pytest.fixture(scope="function")
+def tmp_venv_with_nncf(tmp_path, package_type, venv_type):  # pylint:disable=redefined-outer-name
```

Comment:
This looks very much like its counterpart in torch tests. Can this be refactored to increase code reuse between framework-specific tests?